test/shared/models/audit/FlattenedGenericAuditDetailFixture.scala

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package shared.models.audit
+
+import play.api.libs.json.{JsValue, Json}
+import shared.models.audit.AuditResponseFixture.{auditResponseModelWithBody, auditResponseModelWithErrors}
+import shared.models.auth.UserDetails
+
+object FlattenedGenericAuditDetailFixture {
```

Comment:
Does this definitely all need to be in a separate fixture file?

I guess the other tests in this package do that, but I don't think it's a pattern to be replicated as such. Unless they're being reused elsewhere, I'd suggest just keeping them together in the test class. (Especially the val nino etc)